src/main/java/org/sagebionetworks/bridge/dynamodb/JsonMarshaller.java

```diff
@@ -0,0 +1,45 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import java.io.IOException;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMappingException;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTypeConverter;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+
+/**
+ * Generic JSON marshaller for DynamoDB. This converts the object into JSON for marshalling to DynamoDB. Because
+ * DynamoDB annotations don't work with generics, you'll need to subclass this and fill in getConvertedClass().
+ */
+public abstract class JsonMarshaller<T> implements DynamoDBTypeConverter<String, T> {
```

Comment:
There are no tests directly for this class. It does get some indirect coverage through the exported record info marshaller. But probably worth adding JsonMarshallerTest.